### DISCUSSION TITLE
antdv5的MenuItem如何结合React Router 的Link来实现内容的切换（路由切换）呢？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [ant-design/ant-design#42224] [Q&A] [answered]
antd v5中 [Menu & MenuItem](https://ant.design/components/menu-cn#components-menu-demo-sider-current)：

如何结合React Router 的Link来实现内容的切换（路由切换）呢？

MenuItem貌似对应的属性来接受跳转url，
![image](https://user-images.githubusercontent.com/12727582/236833408-287b6f93-9ebe-41e6-a3c7-57e06f819821.png)

**Comment by BoyYangzai** [comment, marked as answer]
@liaodalin19903 
做了个demo 
https://codesandbox.io/s/modest-https-q9bt59?file=/index.js
cc 稍微有点卡

方案可以参考：
https://blog.csdn.net/liyonghong3333/article/details/124969089
用key+Api跳转也ok
